src/pages/Search/AdvancedSearchFilters.tsx

```diff
@@ -402,7 +418,26 @@ function AdvancedSearchFilters() {
     const allCards = useMemo(() => mergeCardListWithWorkspaceFeeds(workspaceCardFeeds ?? CONST.EMPTY_OBJECT, userCardList, true), [userCardList, workspaceCardFeeds]);
     const taxRates = getAllTaxRates();
     const personalDetails = usePersonalDetails();
-
+    const {canUseLeftHandBar} = usePermissions();
+
+    // If users have access to the leftHandBar beta, then the workspace filter is displyed in the first section of the advanced search filters
+    const typeFiltersKeysWithPermissionCheck = useMemo(
```

Comment:
`withPermissionCheck` sounds super generic and its not easy to know what permission we check.

Change the name to something related to what we are actually implementing. Example: `typeFiltersKeysWith[Optional|Possible]Policy` etc